.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -21,11 +21,11 @@ Provide the generated code, or the error message:
 Translated code here
 ```
 
```

Comment:
Can we add here a section for the Error?
The error can be a compilation error or false code generated or linking error ...